rsocket/internal/Common.h

```diff
@@ -59,6 +59,11 @@ enum class StreamType {
   FNF,
 };
 
+enum class Requester {
```

Comment:
I can see this getting conflated with `RSocketRequester`...  Is there a different name we can use?